__tests__/app.test.js

```diff
@@ -24,10 +24,6 @@ describe('cats routes', () => {
       id: '1',
       name: 'Felix',
       type: 'Tuxedo',
-      url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/0f/Felix_the_cat.svg/200px-Felix_the_cat.svg.png',
-      year: 1892,
-      lives: 3,
-      isSidekick: false,
     };
     expect(res.body).toEqual(felix);
```

Comment:
Be sure to add in the full data of each cat, including id, name, type, url, year, lives, and isSidekick.